### ISSUE TITLE
Export GAML templates

### ISSUE THREAD

**Issue by sriramab** [gama-platform/gama#2396]
GAML templates are very useful especially when teaching new students or for my personal use on complex snippets that i borrow from the forum.  However, it is no possible to export my customized templates to another machine and this negates the very purpose of templates.

i would like to request a possibility to a) export templates b) restore original templates.

thank u
Srirama

**Comment by AlexisDrogoul** [comment]
If you go to Preferences, then "Advanced...", you will find in GAML > Templates a button named "Export...". It will export the selected templates to an xml file. And you can import them later in another instance of GAMA. Does it answer this request ?

**Comment by sriramab** [comment]
Hmm, it does not work for me. DId you try on the standalone GAMA? I feel may be it is working because you launch GAMA via Eclipse as a developer? Both via Model>templates and via Advanced.... does not work for me. I have also tried to create New Templates but still no success. I looked at youtube videos, but no success so far. Anything I am missing? I am not able to use/create self defined templates. 

So how does it work in GAMA? I did ctrl+space and started typing a word (user-defined templates), but nothing is show in the content assist nor am I able to insert any code. (yes, auto insert is enabled for each template)

Thank you
Srirama
